{"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/229/comments/37442163.json"}, "html": {"href": "#!/osrf/srcsim/issues/229#comment-37442163"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/229.json"}}, "type": "issue", "id": 229, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "field computer public IP never came up"}, "content": {"raw": "can we get back the cloudsim hours we lost to this?", "markup": "markdown", "html": "<p>can we get back the cloudsim hours we lost to this?</p>", "type": "rendered"}, "created_on": "2017-06-07T20:29:01.862871+00:00", "user": {"display_name": "dan", "uuid": "{a93bd0ec-35fc-49a5-aa8a-b0b57403c2d0}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Ba93bd0ec-35fc-49a5-aa8a-b0b57403c2d0%7D"}, "html": {"href": "https://bitbucket.org/%7Ba93bd0ec-35fc-49a5-aa8a-b0b57403c2d0%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/dd56d3105d7602d83d5c35a320f6b36cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsD-6.png"}}, "nickname": "dan77062", "type": "user", "account_id": "557058:e0e36946-cb60-4776-8f47-2e9a0285ad35"}, "updated_on": null, "type": "issue_comment", "id": 37442163}